code/datums/emergency_calls/deathsquad.dm

```diff
@@ -49,10 +49,57 @@
 
 	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(to_chat), H, SPAN_BOLD("Objectives: [objectives]")), 1 SECONDS)
 
+/datum/emergency_call/death_lowthreat
```

Comment:
Why not make the lowthreat variants a subtype of the regular one, and just /create_member with weaker gear?